File: 01_Rookie_0800/0001_4A_Watermelon/PROgram0006.py
#########################################################################
#
#  Problem  : 4A - Watermelon
#  Platform : Codeforces
#  Rating   : 800
#  Language : Python 3
#
#########################################################################

#########################################################################
#
#  Function Name : CheckWaterMelon
#  Description   : Checks whether the given watermelon weight can
#                  be divided into two positive even parts.
#
#########################################################################

def CheckWaterMelon(No):
    if ((No >2) and (No % 2 == 0)): 
        return True
    else:
        return False    

#########################################################################
#
#  Function Name : main
#  Description   : Accepts the weight of the watermelon and displays
#                  "YES" if it can be divided into two positive even
#                  parts; otherwise displays "NO".
#
#########################################################################

def main():
    Weight = int(input())

    Ret = CheckWaterMelon(Weight)

    if Ret == True:
        print("YES")
    else:
        print("NO")

if __name__ == "__main__":
    main()

File: 01_Rookie_0800/0002 _71A_Way_To_Long_Words/PROgram0006.js
/////////////////////////////////////////////////////////////////////////
//
//  Problem  : 71A - Way Too Long Words
//  Platform : Codeforces
//  Rating   : 800
//  Language : JavaScript (Node.js)
//
/////////////////////////////////////////////////////////////////////////

/////////////////////////////////////////////////////////////////////////
//
//  Package
//  fs : Used to read input from standard input (stdin).
//
/////////////////////////////////////////////////////////////////////////

const fs = require("fs");

/////////////////////////////////////////////////////////////////////////
//
//  Function Name : PrintWord
//  Description   : Displays the word as it is if its length is
//                  less than or equal to 10. Otherwise, displays
//                  the abbreviated form consisting of the first
//                  character, number of middle characters, and
//                  the last character.
//
/////////////////////////////////////////////////////////////////////////

function PrintWord(word) {
  let len = word.length;

  if (len <= 10) {
    console.log(word);
  } else {
    console.log(word[0] + (len - 2) + word[len - 1]);
  }
}

/////////////////////////////////////////////////////////////////////////
//
//  Function Name : main
//  Description   : Accepts the number of words followed by the
//                  words themselves and displays the required
//                  output for each word.
//
/////////////////////////////////////////////////////////////////////////

function main() {
  let input = fs.readFileSync(0, "utf8").trim().split(/\r?\n/);

  let wordCount = Number(input[0]);

  for (let i = 1; i <= wordCount; i++) {
    PrintWord(input[i]);
  }
}

main();
